types/type.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	DefaultGasLimit = uint64(210000)
+)
+
+type SendTokenRequest struct {
+	Token     string
+	Amount    int64
+	ToAddress string
+}
+
+// TxBroadcastResponse Generic tx response
+type TxBroadcastResponse struct {
```

Comment:
Do we have to define ourselves, can we depend on tendermint definition?

Reply:
yeah can use cosmos' for it.